package.json

```diff
@@ -7,8 +7,10 @@
 	"sideEffects": false,
 	"type": "module",
 	"exports": {
-		"import": "./src/index.js",
-		"require": "./dist/index.cjs",
+    ".": {
+      "import": "./src/index.js",
+      "require": "./dist/index.cjs"
+    },
     "./package.json": "./package.json"
```

Comment:
Are we able to remove this then?

Reply:
I don't understand the question, sorry

Reply:
Are we now able to remove `    "./package.json": "./package.json"`?

Reply:
That's one of the main things I'm adding in this PR. It's essential to achieving the aims of the PR, namely being able to import/require 'node-fetch/package.json' in nodejs 12.7.0+